FiveCalls/FiveCallsUITests/GET-v1-reps.json

```diff
@@ -17,7 +20,7 @@
       "id": "CA-NancyPelosi",
       "name": "Nancy Pelosi",
       "phone": "202-225-4965",
-      "photoURL": "https://repimages.5calls.org/us-house/512x512/nancy-pelosi.jpg",
+      "photoURL": "https://images.5calls.org/house/256/P000197.jpg",
```

Comment:
Looks like the old photo URL's are working anymore. I couldn't find a working URL for Diane Feinstein since she died a few days ago (RIP).

Reply:
Good call, it's been a while since we used these old image urls